Sources/AppCoinsSDK/Domain/Entities/Wallet.swift

```diff
@@ -9,7 +9,7 @@ import Foundation
 
 internal protocol Wallet: Codable {
     
-    func getBalance(currency: Coin, completion: @escaping (Result<Balance, AppcTransactionError>) -> Void)
+    func getBalance(currency: String, completion: @escaping (Result<Balance, AppcTransactionError>) -> Void)
```

Comment:
Use Currency to represent currencies, not String.

Reply:
Done